table/server/underdb/hive/src/main/java/alluxio/table/under/hive/HiveDatabase.java

```diff
@@ -146,72 +152,124 @@ public String getName() {
     }
   }
 
-  private PathTranslator mountAlluxioPaths(Table table, List<Partition> partitions,
-      UdbBypassSpec bypassSpec)
-      throws IOException {
-    String tableName = table.getTableName();
-    AlluxioURI ufsUri;
-    AlluxioURI alluxioUri = mUdbContext.getTableLocation(tableName);
-    String hiveUfsUri = table.getSd().getLocation();
+  /**
+   * Mounts the database to Alluxio filesystem. This method mounts tables and partitions
+   * (if applicable) within this database to Alluxio filesystem.
+   *
+   * @param bypassSpec bypass spec
+   */
+  private void mount(UdbBypassSpec bypassSpec) throws IOException {
+    List<String> tableNames = getTableNames();
```

Comment:
This method will get all tables from udb, it don't respect the `catalog.db.ignore.udb.tables` options

Reply:
This logic is handled by `alluxio.master.table.Database#sync` method. The ignored tables may be mounted along with others, but they will not be visible to Alluxio client.

Reply:
Fixed in 5326f0c146e68f9d3923e2d42d2a2d3089f33d8f